src/router.app.src

```diff
@@ -17,7 +17,6 @@
         helium_proto,
         observer_cli,
         hackney,
-        e2qc,
```

Comment:
I think this needs to stay in the app deps even if we're not pulling it in directly from rebar.

Reply:
it builds and tests fine without re-adding this line.

Output of `rebar3 tree` shows that e2qc is found via another dependency chain, so that might be confirmation enough.

```
└─ router─0.1.0 (project app)
   ├─ blockchain─0.1.0 (git repo)
   │  ├─ e2qc─1.2.1 (git repo)
```

Reply:
Honestly, I've never understood the purpose of this list. I'll re-add it if necessary.